ui/src/components/MultiInputComponent/MultiInputComponent.tsx

```diff
@@ -79,19 +80,20 @@ function MultiInputComponent(props: MultiInputComponentProps) {
         let current = true;
         const abortController = new AbortController();
 
+        const url = referenceName
+            ? generateEndPointUrl(encodeURIComponent(referenceName))
+            : endpointUrl;
```

Comment:
can we leave that 'endpoint' just like that?
Probably yes as we can not create any reasonable validation to it, but just pointing up.

Reply:
yep, we have to. It is a literal URL where we should poke to. And if it is pointing to a attacker domain - nothing we can do

Reply:
As for MSCS
```json
"endpointUrl": "data/indexes",
```
makes `GET /data/indexes`. If we encode this field, it would be `GET data%2Findexes`